modules/ting_covers_ddb/includes/ting_covers_ddb.admin.inc

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @file
+ * Administration interface for the ting_covers_ddb module.
+ */
+
+/**
+ * Administration form for the module.
+ *
+ * @ingroup forms
+ *
+ * @see system_settings_form()
+ */
+function ting_covers_ddb_admin_settings_form($form, &$form_state) {
+  $form['ddb'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('DDB Cover Service'),
+    '#description' => t('Fetch covers form <a href="https://cover.dandigbib.org/">DDB Cover Service</a>. It requires that you have setup and configured the client id/secret in adgangsplatformen module.'),
+    '#tree' => FALSE,
+  );
+
+  $form['ddb']['ting_covers_ddb_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Service URL'),
+    '#description' => t('URL to the Additional Information webservice, e.g. http://cover.dandigbib.org'),
+    '#required' => TRUE,
+    '#default_value' => _ting_covers_ddb_get_url(),
+  );
+
+  $form['ddb']['ting_covers_ddb_enable_logging'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Enable logging'),
+    '#default_value' => variable_get('ting_covers_ddb_enable_logging', FALSE),
+  );
+
+  $form['ddb']['development'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Development settings'),
+    '#description' => t('The module request that you have a valid adgangsplatform setup to get an agency token. For testing you can here set an token (which will expire) if you are on an system without Adgangsplatformen enabled. Should <strong>never</strong> be used in production.'),
```

Comment:
`t('The module request that you have a valid adgangsplatform setup to get an agency token. For testing you can here set an token (which will expire) if you are on an system without Adgangsplatformen enabled. Should <strong>never</strong> be used in production.'),`
->
`t('The module requires that you have a valid "Adgangsplatform" setup to get an agency token. For testing you can manually set an token here if you are on an system without "Adgangsplatformen" enabled. This option should <strong>never</strong> be used in production (Token will not renew once expired). '),`

Reply:
Updated the comment.